src/olympia/addons/indexers.py

```diff
@@ -23,6 +23,8 @@ def get_mapping(cls):
             'properties': {
                 'max': {'type': 'long'},
                 'min': {'type': 'long'},
+                'max_human': {'type': 'string', 'index': 'no'},
+                'min_human': {'type': 'string', 'index': 'no'},
```

Comment:
that sounds good to me, and there's still `max` and `min` in the index if we have the need for it.